external/webserver/http_client.c

```diff
@@ -94,6 +94,8 @@ pthread_addr_t http_handle_client(pthread_addr_t arg)
 			}
 		}
 #endif
+
+#ifdef CONFIG_NETUTILS_WEBSOCKET
```

Comment:
why did you add CONFIG_NETUTILS_WEBSOCKET in here. could you comment the reason?